user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+    
+    
+.. _user-guide-getting-started-log-in:
+
+Log-In
+------
+
+Your administrator will determine this but typically upon your user account creation you will receive an email with your 
+login credentials. In order to log in to the system for the first time, type in the username and password specified in 
+the email. In order to save your credentials on a device, check the *"Remember me on this computer"* box before you log 
+in.
+
+.. caution::
+
+    It is strictly recommended to change your password immediately after the first login. You can learn how to do it in 
+    the :ref:Changing Your Password section `<user-guide-getting-started-profile-password>` below.
+
+If you have forgotten the credentials, click the *"Forgot your password?"* link and follow the wizard to restore them.
+
+
+Log In Using Your Google Credentials
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the Google Single Sign-On capability has been enabled for your organization, you will see the *"login using google"* 
+link at the login screen. This means you can log in using your corporate Gmail account credentials. Usually this 
+capability is enabled for corporate email addresses only, so your personal mailbox credentials won’t do.
+
+
+Log In Using Other Corporate Credentials 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OroCRM can be set up to accept your existing corporate login and password. This is possible if your organization 
+utilizes a common server to share them across multiple applications. This is referred to as LDAP integration. Please 
+check with your system administrator to see if this is the case.
+
+
+.. _user-guide-getting-started-my-user:
+
+Access Your User Page
+---------------------
+
+You can always find a link to your user page in the drop-down menu that resides below your username in the top right 
+corner. This is a fast way to access your user profile, calendar, mailbox, and task list. 
+
+.. image:: ./img/intro/user_menu.png
+
+The following subsection is devoted to the user page that is available with *"My User"* menu. You can find more 
+information about other menu items in the :ref:`User Menu <user-guide-getting-started-user-menu>` document.
+
+.. _user-guide-getting-started-my-user-overview:
+
+My User Page Overview
+^^^^^^^^^^^^^^^^^^^^^
+
+This page provides you the full details of your user profile.
+
+Its *"General"* section contains the basic profile details.
+
+.. image:: ./img/intro/user_view_general.png
+
+The set of information available in the section depends on the 
+:ref:`permissions <user-guide-user-management-permissions-basic>` defined for your user. The information may include:
+
+* Your name and system information such as status (Active), login count, date and time of the last login - in the header
+
+* Your username 
+
+* Your birthday
+
+* Information about your role and place in the company structure, such as your roles in the system and business units 
+  you have access to, your API key (if any)
```

Comment:
Edit: Information about your role and place in the company structure, such as your roles in the system and business units you have access to, as well as your API key (if any)